assets/css/admin.scss

```diff
@@ -6234,3 +6234,25 @@ table.bar_chart {
 		}
 	}
 }
+
+#woocommerce_network_orders {
```

Comment:
Styling issues:

- Table within widget is not 100% width
- TH padding is misaligned with TDs
- Could use some negative margin, or lose the padding on .inside` so the table appears full width and more closely matches other widgets.
- Row has cursor pointer, but is not really clickable.

Reply:
Fixed these.